src/Ship.cpp

```diff
@@ -536,13 +536,15 @@ void Ship::SetAngThrusterState(const vector3d &levels)
 
 vector3d Ship::GetMaxThrust(const vector3d &dir) const
 {
+	size_t thruster_power_cap = 0;
```

Comment:
You might want to add the appropriate pi_lua_generic_pull function in LuaPushPull.h for size_t, because on whatever arch travis compiles, it maps to unsigned long, which doesn't exist, hence the compilation fail.